jcl/src/java.base/share/classes/openj9/internal/criu/InternalCRIUSupport.java

```diff
@@ -64,6 +64,9 @@
  * This API enables the use of CRIU capabilities provided by the OS as well as JVM support for facilitating a successful checkpoint
  * and restore in varying environments.
  */
+/*[IF (17 <= JAVA_SPEC_VERSION) & (JAVA_SPEC_VERSION < 24)]*/
+@SuppressWarnings({ "deprecation", "removal" })
+/*[ENDIF] (17 <= JAVA_SPEC_VERSION) & (JAVA_SPEC_VERSION < 24) */
```

Comment:
```suggestion
/*[IF JAVA_SPEC_VERSION >= 17]*/
@SuppressWarnings({ "deprecation", "removal" })
/*[ENDIF] JAVA_SPEC_VERSION >= 17 */
```
JDK24+ still needs this annotation because this file has `AccessController.doPrivileged()` usages.

Reply:
Fixed.